montecarlo.py

```diff
@@ -114,7 +119,7 @@ def densityCalculator(self,gen,state_d):
             return TypeError("Inappropriate Arguement Type.")
 
     #Monte Carlo Algorithm methods 
-    def simulate(self):
+    def simulateNN(self):

```

Comment:
simulateNN should be renamed, can be done at a later time and with some careful thinking and discussion